staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/managedfields_test.go

```diff
@@ -169,7 +169,7 @@ manager: foo
 operation: Update
 time: "2001-02-03T04:05:06Z"
 `,
-			expected: "{\"manager\":\"foo\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2001-02-03T04:05:06Z\"}",
+			expected: "{\"manager\":\"foo\",\"operation\":\"Update\",\"apiVersion\":\"v1\"}",
```

Comment:
This change omits the time completely in the API? That doesn't seem right? I guess I was expecting the first and last time to get saved for each manager?

Reply:
Also we shouldn't just stop setting a field, we need to remove it from the API if we don't want it.

Reply:
This change doesnt omit the time from the API, it omits the time from the manager identifier which is passed into smd (making it possible for updates from multiple different times to have the same identifier)